internal/cli/runner_install.go

```diff
@@ -0,0 +1,269 @@
+package cli
+
+import (
+	"github.com/hashicorp/waypoint-plugin-sdk/terminal"
+	"github.com/hashicorp/waypoint/internal/clierrors"
+	"github.com/hashicorp/waypoint/internal/installutil"
+	"github.com/hashicorp/waypoint/internal/pkg/flag"
+	"github.com/hashicorp/waypoint/internal/runnerinstall"
+	"github.com/hashicorp/waypoint/pkg/server"
+	pb "github.com/hashicorp/waypoint/pkg/server/gen"
+	"github.com/hashicorp/waypoint/pkg/serverconfig"
+	"github.com/posener/complete"
+	empty "google.golang.org/protobuf/types/known/emptypb"
+	"sort"
+	"strings"
+)
+
+type RunnerInstallCommand struct {
+	*baseCommand
+
+	platform              []string `hcl:"platform,required"`
+	adopt                 bool     `hcl:"adopt,optional"`
+	serverUrl             string   `hcl:"server_url,required"`
+	id                    string   `hcl:"id,optional"`
+	runnerProfileOdrImage string   `hcl:"id,optional"`
+	serverTls             bool     `hcl:"server_tls,optional"`
+	serverTlsSkipVerify   bool     `hcl:"server_tls_skip_verify,optional"`
+	serverRequireAuth     bool     `hcl:"server_require_auth,optional"`
+}
+
+func (c *RunnerInstallCommand) AutocompleteArgs() complete.Predictor {
+	return complete.PredictNothing
+}
+
+func (c *RunnerInstallCommand) AutocompleteFlags() complete.Flags {
+	return c.Flags().Completions()
+}
+
+func (c *RunnerInstallCommand) Flags() *flag.Sets {
+	return c.flagSet(0, func(set *flag.Sets) {
+		f := set.NewSet("Command Options")
+
+		var runnerPlatforms []string
+		for platformName := range runnerinstall.Platforms {
+			runnerPlatforms = append(runnerPlatforms, platformName)
+		}
+
+		f.EnumVar(&flag.EnumVar{
+			Name:   "platform",
+			Usage:  "Platform to install the Waypoint runner into.",
+			Values: runnerPlatforms,
+			Target: &c.platform,
+		})
+
+		f.StringVar(&flag.StringVar{
+			Name:   "server-addr",
+			Usage:  "Address of the Waypoint server.",
+			EnvVar: "WAYPOINT_ADDR",
+			Target: &c.serverUrl,
+		})
+
+		f.StringVar(&flag.StringVar{
+			Name:    "odr-image",
+			Usage:   "Docker image for the on-demand runners. If not specified, it defaults to 'hashicorp/waypoint-odr:latest'.",
```

Comment:
No need for the default verbage. If you notice, the flag package actually adds that for you if you set a default on a flag!